### PR TITLE
Fix: Advanced columns - Column not stacking on tablet & mobile devices.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/a0Bsp9IW/255-fix-advanced-columns-column-not-stacking-on-tablet-mobile-devices

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop advanced column with 2 child with 50% width.
- Select option stack on > tablet and check is stacking on tablet and mobile both.
- Select option stack on > mobile. It should stack on mobile devices only.
- Select option stack on > none.  It should not stack on tablet & mobile devices.  

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/columns/edit.js [hunk 1]
```diff
@@ -659,12 +659,12 @@ class UAGBColumns extends Component {
...
 						/>
 						<ToggleControl
-							label={ __( "Reverse Columns (Tablet)", 'ultimate-addons-for-gutenberg' ) }
+							label={ __( "Reverse Columns (Tablet & Mobile)", 'ultimate-addons-for-gutenberg' ) }
 							checked={ reverseTablet }
 							onChange={ ( value ) => setAttributes( { reverseTablet: ! reverseTablet } ) }
 						/>
 						<ToggleControl
-							label={ __( "Reverse Columns (Mobile)", 'ultimate-addons-for-gutenberg' ) }
+							label={ __( "Reverse Columns (Mobile only)", 'ultimate-addons-for-gutenberg' ) }
```
Comment (RajkiranBagal) [inline src/blocks/columns/edit.js:667]: Remove only from here.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: readme.txt [hunk 2]
```diff
@@ -168,6 +168,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
+= 1.22.1.1 =
+* Fix: Advanced Column - Column not stacking on tablet & mobile devices.
```
Comment (RajkiranBagal) [inline readme.txt:172]: Rename `Advanced Column` to `Advanced Columns`

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (AbhijitNage123) [reply readme.txt]: Done

File: src/blocks/columns/edit.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/columns/edit.js]: Done

File: src/blocks/columns/edit.js [hunk 3]
```diff
@@ -596,8 +596,8 @@ class UAGBColumns extends Component {
 							value={ stack }
 							options={ [
 								{ value: "none", label: __( "None", 'ultimate-addons-for-gutenberg' ) },
-								{ value: "tablet", label: __( "Tablet", 'ultimate-addons-for-gutenberg' ) },
-								{ value: "mobile", label: __( "Mobile", 'ultimate-addons-for-gutenberg' ) },
+								{ value: "tablet", label: __( "Tablet & Mobile", 'ultimate-addons-for-gutenberg' ) },
+								{ value: "mobile", label: __( "Mobile only", 'ultimate-addons-for-gutenberg' ) },
```
Comment (RajkiranBagal) [inline src/blocks/columns/edit.js:600]: Remove only from here.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**